content/cloud-docs/api-docs/variable-sets.mdx

```diff
@@ -128,12 +128,24 @@ curl \
 
 ## Update a Variable Set
 
-`PUT/PATCH varsets/:varset_id`
+`PATCH varsets/:varset_id`
 
 | Parameter    | Description         |
 | ------------ | ------------------- |
 | `:varset_id` | The variable set ID |
 
+### Request Body
+
+Properties without a default value are required.
+
+| Key path                        | Type    | Default | Description                                                                                                                             |
+| ------------------------------- | ------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| `data.name`                     | string  |         | The name of the variable set.                                                                                                           |
+| `data.description`              | string  | `""`    | Text displayed in the UI to contextualize the variable set and its purpose.                                                             |
+| `data.global`                   | boolean | `false` | When true, Terraform Cloud automatically applies the variable set to all current and future workspaces in the organization.             |
```

Comment:
This could use an explanation of how `attributes.global` and `relationships.workspaces` interact. I knew this at one point when we were reviewing a go-tfe PR, but I can't remember and wouldn't be able to derive it from the current docs without doing an experiment.